src/app/components/create-wallet-form/create-wallet-form.component.ts

```diff
@@ -20,23 +19,40 @@ export class CreateWalletFormComponent implements OnInit {
   handleSubmit(form: NgForm): void {
     this.markFormAsDirty(form);
     if (form.valid) {
-      this.createWallet(form.value.walletId, form.value.walletPassphrase);
+      this.createWallet(
+        form.value.walletId,
+        form.value.walletPassphrase,
+        form.value.walletMnemonics
+      );
+    } else {
+      alert("create-wallet form not valid");
     }
   }
 
   markFormAsDirty(form: NgForm) {
-    Object.keys(form.controls).forEach(field => {
+    Object.keys(form.controls).forEach((field) => {
       form.controls[field].markAsDirty();
     });
   }
 
-  createWallet(id: string, passphrase: string): void {
-    this.walletService.addWallet(id, passphrase).subscribe(
-      () => {
+  createWallet(id: string, passphrase: string, mnemonics: string): void {
+    this.walletService.addWallet(id, passphrase, mnemonics).subscribe(
+      (a) => {
         this.walletService.syncWalletList();
+
+        if (a["result"]) {
+          if (mnemonics != undefined && mnemonics.length > 0) {
+          } else {
+            let mnemonics = a["result"][1];
+            alert("mnemonics= " + mnemonics);
```

Comment:
is this JS alert box or some angular modal UI alert?

Reply:
i'll find it

Reply:
For modal you can take a reference of
https://github.com/crypto-com/sample-chain-wallet/blob/f808201101a3113c5e9cee78fc59ba6c8739c300/src/app/components/wallet-info/wallet-info.component.html#L141
https://github.com/crypto-com/sample-chain-wallet/blob/f808201101a3113c5e9cee78fc59ba6c8739c300/src/app/components/wallet-info/wallet-info.component.ts#L38